pages/data-design/quick-start.mdx

```diff
@@ -0,0 +1,116 @@
+
+> If you already have an Avo workspace, you can skip this step.
+
+### Step 2: Import your existing tracking plan
+
+If you have an existing tracking plan that you want to import, you can do so by clicking the **Import** button in the workspace's settings.
+<br/>
+<center>
+  <img
+    src={require('../../images/audit/import-button.png')}
+    alt="Import button in Avo workspace"
+    width="450"
+  />
+</center>
+<br/>
+<br/>
+
+You can import a tracking plan from any of the following sources: Excel or Google Sheets (.csv), Mixpanel Lexicon or Amplitude Govern. Learn more about the available import formats in the <Link passHref href="/workspace/tracking-plan/importing" passHref><a>importing docs</a></Link>. If you have your tracking plan on a format that's currently not supported by the tracking plan importer, <Link passHref href="/workspace/tracking-plan/importing" passHref><a>reach out</a></Link> to us and we'll import it for you.
+
+> If you don't have an existing tracking plan, or if you just want to start fresh, you can skip this step. No worries, you can always import your existing tracking plan later.
+
+### Step 3: Create a new branch and start making changes
+
+If you are making changes to your tracking plan in Avo, we always recommend creating a branch first – no matter how small your change might be. That allows you to make suggestions in isolation from the source of truth on your main branch and get feedback from your team.
+
+Open a new branch by clicking the three line icon beside where it says "main" in the left sidebar. Then you are prompted to open an existing branch or creating a new one.
+
+<center>
+<img
+  src={require('../../images/best-practices/getting-started-to-design-data/opening-a-branch.gif')}
+  alt="Opening a branch"
+  width="450"
+/>
+</center>
+<br />
+<br />
+
+When you've created the branch, navigate to the *Events* tab in the tracking plan and create a new event by clicking the *+ Add Event* button.
+
+<center>
+<img
+  src={require('../../images/best-practices/getting-started-to-design-data/create-event.gif')}
+  alt="Creating a new event"
+  width="450"
+/>
+</center>
+
+### Step 4: Invite a colleague and ask them for a review
+
+Now we're ready to share our suggestions with our team, and ask them for input. Go to your workspace settings and invite a team member. Workspace members can be one of admin, editor, comment only or view only. To learn more about members and roles, check out our <Link passHref href="/workspace/members">members docs</Link>.
+
+<center>
+<img
+  width="450"
+  src={require('../../images/workspace/members/accessMembers.png')}
+  alt="Accessing workspace settings"
+/>
+</center>
+
+<br />
+<br />
+
+To review the changes you've made on your branch, click the *View changes* button in the workspace sidebar. There you can see all changes made to the tracking plan on this branch.
```

Comment:
Maybe sneak in something about the Workbench here as well?

Reply:
Riight, added in 5906593. Looks like we didn't ship any docs with the workbench, would be nice to link to those as well. [Added a task here](https://app.asana.com/0/1200571264461528/1200637169007883) for adding a Workbench doc to the **Tracking Plan Management** > **The Avo Tracking Plan** section